src/services/getFrameKmsService.ts

```diff
@@ -35,7 +35,7 @@ const promiseExec = promisify(exec);
 
 
 async function postFrameKmRequest(framekm_name: string) {
-    const url = `${BASE_URL}/framekm/request/1.8`;
+    const url = `${BASE_URL}/framekm/request/${FRAMEKM_VERSION}`;
```

Comment:
Very small callout:
I'd probably better use FrameKM version from metadata, cause of a single scenario:
- framekm collected with version N but not uploaded yet
- the firmware updated to N+1 version
- postFrameKmRequest will then try to upload FrameKM vN using URL for N+1

Reply:
Changed to check the metadata version